mantidimaging/core/convert/__init__.py

```diff
@@ -23,4 +23,7 @@
 # Code Documentation is available at: <http://doxygen.mantidproject.org>
 
 __all__ = ['convert']
+
 from . import convert  # noqa: F401, F403
+
+del absolute_import, division, print_function
```

Comment:
What's the reason we need to delete this here?

Reply:
It's done to prevent those symbols appearing when iPython auto-completion is used.

Reply:
I see. I hadn't expanded the diff so I couldn't see them originally so I wondered how they had crept in.